code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -1706,7 +1706,14 @@ datum
 					return
 				if (method == TOUCH)
 					. = 0 // for depleting fluid pools
+				if (!ON_COOLDOWN(M, "ants_scream", 15 SECONDS)) //lets make it less spammy
```

Comment:
Also I authored the original scream spam reduction change, but 15 seconds is HARSH. Drop to 10 imo, or even 5.